lxd/networks.go

```diff
@@ -133,9 +133,8 @@ func networksPost(d *Daemon, r *http.Request) response.Response {
 	resp := response.SyncResponseLocation(true, nil, url)
 
 	if isClusterNotification(r) {
-		// This is an internal request which triggers the actual
-		// creation of the network across all nodes, after they have
-		// been previously defined.
+		// This is an internal request which triggers the actual creation of the network across all nodes
+		// after they have been previously defined.
 		err = doNetworksCreate(d, req, true)
```

Comment:
@stgraber I believe this was a bug before, because previously the 3rd argument here was called `withDatabase` and meant that the DB record was removed on error, but it is wrapped in a `isClusterNotification` block, suggesting it should have been `false`. However now the argument is `clusterNotification`, the existing `true` is fine and means database record removal won't occur on error.